composer.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "mediawiki/rdfio",
```

Comment:
Once merged, please make sure to register with [0].

[0] https://packagist.org/